src/api/browse.ts

```diff
@@ -1,5 +1,28 @@
 import { searchResponseModel, datasetEmbeddedModel, datasetSummaryModel } from "../models/dataset";
 import { facetFilterModel } from "../models/facets";
+import authService from "../services/auth";
+
+const SEARCH_URL = process.env.REACT_APP_SVC_SEARCH_URL;
+const REPOSITORY_URL = process.env.REACT_APP_SVC_REPOSITORY_URL;
+const CLIENT_URL = process.env.REACT_APP_CLIENT_URL;
+
+const getHeaders = async () => {
```

Comment:
We can move this method to the utils and we can introduce one more for the POST methods.

Reply:
I have now added a simplified fetch wrapper method to utils that adds all the necessary headers automatically.